### PR TITLE
Update clean command to also work on windows

### PR DESCRIPTION
**Pull request by Uhuh**

### CODE REVIEW

**Comment by user404d** [comment]
So the problem isn't actually with anything you've done, there's an issue with our `colisee-lib` module.

File: package.json [hunk 1]
```diff
@@ -7,7 +7,9 @@
     "start": "node src/app.js",
     "start-prod": "NODE_ENV=production node src/main.js",
     "start-dev": "npm run build && node src/main.js",
-    "clean": "find src test \\( -name *.js -or -name *.js.map \\) -exec rm {} +",
```
Comment (russleyshaw) [inline package.json]: You still need a primary clean command that invokes all other cleans

File: package.json [hunk 2]
```diff
@@ -31,6 +33,7 @@
     "@types/cors": "^2.8.0",
     "@types/dotenv": "^4.0.1",
     "@types/express": "^4.0.35",
+    "@types/knex": "0.0.61",
```
Comment (russleyshaw) [inline package.json]: This is not right.

File: test/integration/app.ts [hunk 3]
```diff
@@ -1,7 +1,7 @@
-import { expect } from 'chai';
+import { db } from "@siggame/colisee-lib";
+import { expect } from "chai";
 
-import * as db from "../../src/common/db";
 import { App } from "../../src/app";
 
-describe('App', function() {
+describe('App', function () {
```
Comment (russleyshaw) [inline test/integration/app.ts]: why were these spaces added? Is this part of the tslint schema?

**Pull request merged**